ui_framework/components/local_nav/_local_search.scss

```diff
@@ -28,6 +28,10 @@
   }
 }
 
+.kuiLocalLuceneSearchInput:focus {
+  padding-right: 13em;
+}
```

Comment:
Normally I'd agree, but the 13em padding is really tightly coupled with the length of the string that goes in there, i.e. "Uses Lucene query syntax" is 13em wide.